opentelemetry-api/src/logs/record.rs

```diff
@@ -327,7 +327,7 @@ impl LogRecordBuilder {
     }
 
     /// Assign attributes, overriding previously set attributes
-    pub fn with_attributes(self, attributes: OrderMap<Key, AnyValue>) -> Self {
+    pub fn with_attributes(self, attributes: Vec<(Key, AnyValue)>) -> Self {
```

Comment:
Since user will now handle deduplication themselves. Should we call it call in comments explicitly so that for users needs unique attributes can handle it themselves

Reply:
Good point, have added the comment for `LogRecordBuilder::with_attributes` and `LogRecordBuilder::with_attribute` methods.